### ISSUE TITLE
peptide coverage

### ISSUE THREAD

**Issue by QWWHHH** [vdemichev/DiaNN#797]
Hi 
whether the exported results from DIA_NN contain information about peptide coverage?
if not, how can I use the output files to calculate the peptide coverage? at the same time, if there are any methods to avoid computation errors caused by duplicate peptides while calculating the coverage.
many thanks

**Comment by vdemichev** [comment]
No, need to use some R or Python package that loads a FASTA database to calculate it with some script.
